tests/browser_test.py

```diff
@@ -64,6 +64,7 @@ async def test_review_signoff(
         + f"#/buckets/{source_bucket}/collections/{source_collection}/simple-review"
     )
     selenium.refresh()
+    time.sleep(1)  # preventing hooks race condition that occurs on github
```

Comment:
Can we try using this Selenium method for explicitly waiting? 

https://www.selenium.dev/documentation/webdriver/waits/#explicit-waits